daemon/response.go

```diff
@@ -149,6 +149,8 @@ const (
 
 	errorKindNetworkTimeout      = errorKind("network-timeout")
 	errorKindInterfacesUnchanged = errorKind("interfaces-unchanged")
+
+	errorKindConfigNoSuchOption = errorKind("config-no-such-option")
```

Comment:
"option-not-found" seems better here, per pattern established above (snap-not-found, app-not-found)